dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1086,4 +1085,10 @@ internal class RumViewScopeTest {
     }
 
     // endregion
+
+    private fun mockEvent(): RumRawEvent {
```

Comment:
Hmm maybe we should extract this as it is being duplicated in all these classes ?

Reply:
Nope, it's just a local test definition. I don't want to overengineer things for 2 lines.